### DISCUSSION TITLE
PersistentVolumeClaims -> Capacity and Used (negative value)

### DISCUSSION THREAD

**Discussion by ImperrattoR** [okd-project/okd#1274] [Q&A]
Hey, I need some help.
I want to know how Capacity and Used works on PersistentVolumeClaims (console view).

For example I have created PVC for prometheus (40Gb per pod).
Right after I have created PVC I can see that used data is much bigger than capacity.
In prometheus PVC I have available -70,6GiB? Why it is like that (negative value)?

I see that kind of relationship between Used and Capacity on a lot of persisten volume claims.

**Comment by vrutkovs** [comment]
Check your CSI driver documentation to find out why this happens
